build-system/compile/sources.js

```diff
@@ -150,7 +149,6 @@ const CLOSURE_SRC_GLOBS = [
  */
 const THIRD_PARTY_TRANSFORM_GLOBS = [
   // JSX syntax should undergo usual transforms
-  'third_party/optimized-svg-icons/social-share-svgs.js',
```

Comment:
@krdwan Curious about why `THIRD_PARTY_TRANSFORM_GLOBS` was left here an empty list. Any reason to keep it around?

Reply:
@rsimha I wasn't sure if this was a generic part of the AMP build system and that it might be updated to be non-empty in the future.  From the perspective of this PR and the `social-share` component, there's no reason to keep it around.